Client/Helpers/FxALoginHelper.swift

```diff
@@ -54,15 +54,21 @@ class FxALoginHelper {
     // to changing of user settings and push notifications.
     func application(_ application: UIApplication, didLoadProfile profile: Profile) {
         self.profile = profile
+        self.account = profile.getAccount()
 
-        guard let account = profile.getAccount() else {
+        guard let account = self.account else {
             // There's no account, no further action.
             return loginDidFail()
```

Comment:
Looks like `loginDidFail` will call `delegate?.accountLoginDidFail` which is only implemented by the FxA ContentViewController. So that will be a no-op during app startup.

Should there be a `return` here? If there is no account at all, can we proceed at all at this point?